src/app/pages/Search/Search.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Search from './Search';
+
+export default {
+  title: 'Pages/Search',
+  component: Search,
+
+  parameters: {
+    backgrounds: {
+      default: 'dark',
+    },
+    layout: 'fullscreen',
+  },
+};
+
+export const SearchPage = (): JSX.Element => <Search />;
```

Comment:
We decided, that we don't really need stories for our pages. Please remove the file